pkg/state/state_ofed.go

```diff
@@ -735,3 +753,27 @@ func (s *stateOFED) handleRepoConfig(
 	}
 	return nil
 }
+
+// getOCPDriverToolkitImage gets the DTK ImageStream and return the DTK image according to RHCOS version
+func (s *stateOFED) getOCPDriverToolkitImage(ctx context.Context, rhcosVersion string) (string, error) {
```

Comment:
rhcosVersion -> ostreeVersion 

this exists also for rhel (OSTREE_VERSION) i assume.
and we should support dtk on rhel as well right ?

`OSTREE_VERSION="412.86.202301311551-0"`

Reply:
Renamed